spec/motion-support/core_ext/object/inclusion_spec.rb

```diff
@@ -31,4 +31,18 @@ def include?(obj)
       0.in?(LifeUniverseAndEverything.new).should == false
     end
   end
+
+  describe "presence_in" do
+    it "should return self when included" do
+      "stuff".presence_in(%w( lots of stuff )).should == "stuff"
+    end
+
+    it "should return nil when not included" do
+      "stuff".presence_in(%w( lots of crap )).should == nil
+    end
+
+    it "should raise ArgumentError when arg does not respond to `include?`" do
+      lambda { 1.presence_in(1) }.should.raise ArgumentError
+    end
+  end
```

Comment:
For the implementation, corresponding commit is 2d2776e2469f53f27929c5dac2506e9cd10a5449

I removed work around on `#in?`.
I checked the commit logs and found that this work around was for testing.
Removing the work around did not cause any problem on the test any more, so that I made it as it is.

Additionally, I imported `#presence_in` from the current version of `active_support`.
https://github.com/rails/rails/blob/44cff3045c5d98fb368e86949dd0973ee768c409/activesupport/lib/active_support/core_ext/object/inclusion.rb